Makefile

```diff
@@ -1,4 +1,4 @@
-version = $(shell sh ./version)
+version = ${VERSION}
```

Comment:
using a version env variable instead of a hardcoded version is much easier, especially when building snapshots of a commit